etc/profile-a-l/evolution.profile

```diff
@@ -40,7 +72,25 @@ novideo
 protocol unix,inet,inet6
 seccomp
 shell none
+tracelog
 
+disable-mnt
+# Add "gpg,gpg2,gpg-agent,pinentry-curses,pinentry-emacs,pinentry-fltk,pinentry-gnome3,pinentry-gtk,pinentry-gtk2,pinentry-gtk-2,pinentry-qt,pinentry-qt4,pinentry-tty,pinentry-x2go,pinentry-kwallet" for gpg
+private-bin evolution
+private-cache
 private-dev
+private-etc alternatives,ca-certificates,crypto-policies,dconf,fonts,gcrypt,gtk-2.0,gtk-3.0,groups,hostname,hosts,mailname,passwd,pki,resolv.conf,selinux,ssl,xdg
 private-tmp
+writable-run-user
 writable-var
+
+dbus-user filter
+dbus-user.own org.gnome.Evolution
```

Comment:
The flatpak has also
```
org.gnome.keyring.SystemPrompter=talk
org.gnome.OnlineAccounts=talk
```
I'm not sure if they are needed.

Reply:
The prompter is for the password box I think, in this regard seahorse needs to be fixed, exporting secret key doesn't work with current dbus-filters, it's probably x.x.x.internalprompter something. I don't know about online accounts...